mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -13,36 +16,48 @@ CLASS aij net/minecraft/entity/Entity
 	FIELD O age I
 	FIELD P insideWater Z
 	FIELD Q waterHeight D
+	FIELD R wet Z
```

Comment:
This is much more than wet. This field is true if the entity has their head in water, I would recommend `submergedInWater`.

Reply:
`submergedInWater` is such a mouth-full though. I'd more likely just called it `submerged`.

Reply:
I've decided to change this it `inWater` to be in line with `inLava`.